TASVideos/wwwroot/css/partials/_darkmode.scss

```diff
@@ -110,4 +110,12 @@ body {
 	&-white {
 		filter: none;
 	}
-}
\ No newline at end of file
+}
+
+.btn-info {
```

Comment:
The dark mode file should minimize custom code and focus on variable overrides, if you want the info text to always be light in both themes I would define another variable in the bootstrap_overrides to use like info-text instead of bs-white so it doesn't get changed by the theme. Alternatively since you're already changing the colors you could make the info blue light enough for black text to be wcag AA compliant on light theme and use the same color as text-dark did before

Reply:
I think having the text be white across all of the buttons there would look nice and potentially be more readable, I can go ahead and use a new variable as you suggested.